chainerx_cc/chainerx/cuda/numeric.cuh

```diff
@@ -77,5 +77,22 @@ CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Log10, std::log10)
 CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Sqrt, std::sqrt)
 CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Fabs, std::fabs)
 
+template <typename T>
+__device__ inline T Power(T x1, T x2) {
+    return T{rint(pow(static_cast<double>(x1), static_cast<double>(x2)))};
```

Comment:
How about implementing with loop with O(log(x2)) times?
`std::pow` sometime causes a little error.
```py
>>> numpy.power(numpy.array([123456789]), numpy.array([2]))
array([15241578750190521])
>>> chainerx.power(chainerx.array([123456789]), chainerx.array([2]))
array([15241578750190520], shape=(1,), dtype=int64, device='native:0')
```

Reply:
Thanks for the suggestion, fixed accordingly.